docs/content/tutorials/cluster.md

```diff
@@ -0,0 +1,305 @@
+---
+layout: doc_page
+---
+
+# Clustering
+
+Druid is designed to be deployed as a scalable, fault-tolerant cluster.
+
+In this document, we'll set up a simple cluster and discuss how it can be further configured to meet
+your needs. This simple cluster will feature scalable, fault-tolerant servers for Historicals and MiddleManagers, and a single
+coordination server to host the Coordinator and Overlord processes. In production, we recommend deploying Coordinators and Overlords in a fault-tolerant
+configuration as well.
+
+## Select hardware
+
+The Coordinator and Overlord processes can be co-located on a single server that is responsible for handling the metadata and coordination needs of your cluster. 
+The equivalent of an AWS [m3.xlarge](https://aws.amazon.com/ec2/instance-types/#M3) is sufficient for most clusters. This
+hardware offers:
+
+- 4 vCPUs
+- 15 GB RAM
+- 80 GB SSD storage
+
+Historicals and MiddleManagers can be colocated on a single server to handle the actual data in your cluster. These servers benefit greatly from CPU, RAM,
```

Comment:
You don't want to scare people with the complexity of Druid right away. I think it is okay to gradually introduce it.